### ISSUE TITLE
Is PocoProject.org down?

### ISSUE THREAD

**Issue by htartisan** [pocoproject/poco#3120]
What's up with the PocoProject.org web site?  I haven't been able to access that website (any pages on PocoProject.org domain) for weeks.

**Comment by mikallos** [comment]
Works fine for me. Try from another device/network

**Comment by aleks-f** [comment]
No, it is not down.

**Comment by htartisan** [comment]
[Image]
________________________________

**Comment by intensifi** [comment]
It's down for me too via cable modem.  I can get to it when on cellular data (Verizon).  Comcast Advanced Security has an issue with the certificate.  This started approx 2 weeks ago.

I found this article.  Hope it is helpful for "fixing" the new server certificate Gunter told me about via email:

https://www.thesslstore.com/blog/ssl_error_rx_record_too_long/

**Comment by htartisan** [comment]
Yeah, I can't access it from any device on my home (cablemodem) network, or from my cel network.

________________________________

**Comment by obiltschnig** [comment]
@htartisan What is the error message you are seeing?

**Comment by obiltschnig** [comment]
We had some issue a few weeks back where for some reason pocoproject.org landed on the block list of a few anti malware companies (according to https://www.virustotal.com/gui/url/7432a279b815b5ec669cc1db399e05d7ae656203cf0a62285237eb520d0b4dd8/detection). This has been fixed in the meantime, but it could be that some lax providers don't update their lists regularly.

**Comment by obiltschnig** [comment]
Pretty sure it's not an SSL/TLS issue. pocoproject.org has A+ on https://www.ssllabs.com/ssltest/analyze.html?d=pocoproject.org

**Comment by tdietel** [comment]
I have the same issue, and it seems to be connected to IPv6. I am running macOS Catalina 10.15.7, and I have an IPv6 tunnel running. 

When I tried to get the web page with curl, I got an error message:
$ curl https://pocoproject.org
curl: (35) LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to pocoproject.org:443

I found a suggestion to disable IPv6: https://stackoverflow.com/a/51544596

With IPv6 off, I managed to access the site; and even after re-enabling IPv6 it still seems to work. 

Maybe this helps to trace down the error.
